website/pages/docs/upgrading/upgrade-to-1.5.0.mdx

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.5.0 - Guides
+sidebar_title: Upgrade to 1.5.0
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.5.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.4.x compared to 1.5.0. Please read it carefully.
+
+## Raft Configuration
+
+A new Raft configuration value, `max_entry_size`, has been introduced. This value
+limits the size in bytes for a Raft K/V entry. It applies to both put operations and
+transactions. Any put or transaction operation exceeding this configuration value
+will cause the respective operation to fail. The default value for this
+configuration is 1MiB.
+
+In addition, a new metric has been introduced, `vault.raft-storage.entrySize`,
```

Comment:
The upgrade page usually only contains changes around breaking or deprecated behavior. Although this bit might be useful, I wonder if it's okay to leave it out since that's also in the metrics doc page and in the changelog.

Reply:
Thoughts @mladlow?

Reply:
If I were reading this as an operator, my first question would be "will this change cause problems if I upgrade?" and I'd want to know how to answer that question. So, I think it's a nice convenience for someone like that to know where they can go without having to find the right metric. It can be a little difficult to find the metric you're looking for on the metrics page, I have found.

Reply:
Maybe we should put the metric into 1.4.3 then? That way someone upgrading from that version or greater can check the metric before upgrading

Reply:
Oh, that would be good. @alexanderbez would you mind making a backport PR for the metric to release/1.4.x?

Reply:
Yeah, good idea. Will create (my first) backport PR 👍

Reply:
https://github.com/hashicorp/vault/pull/9105